api/librdb-api.h

```diff
@@ -124,35 +124,35 @@ typedef enum RdbBulkAllocType {
 
 typedef struct RdbKeyInfo {
     long long expiretime;
-    uint64_t lru_idle;
-    int lfu_freq;
+    uint64_t lru_idle; /* TODO: support lru_idle */
+    int lfu_freq; /* TODO: support lfu_freq */
     int opcode;
 } RdbKeyInfo;
 
+typedef struct RdbStreamID {
+    uint64_t ms;
+    uint64_t seq;
+} RdbStreamID;
+
+typedef struct RdbStreamMeta {
```

Comment:
if we wanna have some ABI flexibility (compiling with one version and using another), we probably need a version field.

Reply:
Added semver.

Reply:
i meant adding a version field as a first field in the struct (like we do in some module APIs in redis).
what you did could be sufficient, but it's a little bit informal / awkward to check the library version to know what's in each struct.